bigchaindb/tendermint/utils.py

```diff
@@ -12,3 +13,11 @@ def decode_transaction(raw):
     """Decode a transaction from Base64 to a dict."""
 
     return json.loads(raw.decode('utf8'))
+
+
+def calculate_hash(key_list):
```

Comment:
Does Tendermint expect a SHA-256 hash (i.e. a SHA-2 hash), or can we give it a SHA3-256 hash instead? BigchainDB usually uses SHA3-256 hashes. For example, the ID of a BigchainDB transaction is a SHA3-256 hash (represented as a 64-character hex string. 256 bits = 32 bytes and each byte can be represented by two hex characters).

Reply:
Tendermint doesn't seem to specify any such contraint, so I have moved to SHA3-256.